bigbluebutton-html5/imports/ui/components/video-provider/video-button/service.js

```diff
@@ -4,7 +4,7 @@ import Auth from '/imports/ui/services/auth';
 import Users from '/imports/api/users/';
 import VideoService from '../service';
 
-const baseName = Meteor.settings.public.app.basename;
+const baseName = Meteor.settings.public.app.cdn || Meteor.settings.public.app.basename;
```

Comment:
My understanding was that the basename would persist even if we were using a CDN for some files.

https://my_service.com/<basename>/join?sessionToken=dsvfiadrmtqse6of (what I see when I log in)
would have pulled https://the_cdn_i_use.com/staticFile001.png

So I am not sure why we pick one or the other on line 7

Reply:
The `baseName` variable would contain the default value `/html5client` or the provided value for the CDN URL if provide. The CDN value has always a precedence if not empty  over the `baseName`.

Reply:
I did not realize at first that you provide {domain + baseName} in `env` `export CDN_URL=https://mybbbserver/html5client`
Now I see what you mean.